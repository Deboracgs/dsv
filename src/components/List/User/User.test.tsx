import { render } from "@testing-library/react";
import WrapperMock from "../../../utils/wrapperMock";
import ListUser from "./User";

describe("List User Component", () => {

  const rendered = () =>
    render(
      <WrapperMock>
        <ListUser  />
      </WrapperMock>
    );

  it("render", () => {
    const { getByTestId, asFragment } = rendered();

    const element = getByTestId(`list-user`);

    expect(element).toBeDefined();
    expect(asFragment()).toMatchSnapshot();
  });

});
