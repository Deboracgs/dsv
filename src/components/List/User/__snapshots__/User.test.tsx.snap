// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List User Component render 1`] = `
<DocumentFragment>
  <div
    data-testid="list-user"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
    />
  </div>
</DocumentFragment>
`;
